lib/tezos/lib/assets/download-snapshot.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set +e
+
+source /etc/environment
+
+echo "Downloading TZ snapshot from $TZ_SNAPSHOTS_URI."
+
+
+TZ_SNAPSHOTS_FILE_NAME=snapshot
+
+mkdir ~/tezos-snapshots
+wget -O ~/tezos-snapshots/$TZ_SNAPSHOTS_FILE_NAME $TZ_SNAPSHOTS_URI
```

Comment:
What happens if the download is not successful?

Reply:
It just sync from scratch instead of using a snapshot